mne/viz/_brain/_brain.py

```diff
@@ -2580,7 +2578,11 @@ def save_image(self, filename, mode='rgb'):
         mode : str
             Either 'rgb' or 'rgba' for values to return.
         """
-        self._renderer.screenshot(mode=mode, filename=filename)
+        default_name = _generate_default_filename(".png")
+        filename = default_name if filename is None else filename
+        from ..utils import _save_ndarray_img
+        _save_ndarray_img(
+            filename, self.screenshot(mode=mode, time_viewer=True))
```

Comment:
Do we even need  a `time_viewer` op.tion in `screenshot` anymore? Nothing changes with the 3D rendering with this anymore because all controls are in Qt or ipywidgets, right?

But we probably *should* add a `show_traces` argument. But both of these could be in a quick follow-up PR I think.

Reply:
You're right, I don't think this parameter is necessary anymore in here